db/migrate/20160416163754_add_is_translated_check_to_questions.rb

```diff
@@ -0,0 +1,9 @@
+class AddIsTranslatedCheckToQuestions < ActiveRecord::Migration
```

Comment:
Our schema file's version is `20160706031054`, so I think this migration needs to be regenerated with a newer date in order to run.